kong/runloop/balancer/round_robin.lua

```diff
@@ -56,11 +55,15 @@ function roundrobin_algorithm:afterHostUpdate()
   end
 
   -- add all addresses to the wheel
+  local new_wheel = {}
+  local idx = 1
+
   for _, targets in ipairs(targets) do
```

Comment:
There is a typo here. In L61, the first variable should be "target" instead of "targets".

Reply:
Good Catch! Fixed.